android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java

```diff
@@ -1,7 +1,7 @@
 package ir.sohreco.androidfilechooser;
 
-public class ExternalStorageNotAvailableException extends Exception {
-    public ExternalStorageNotAvailableException() {
+class ExternalStorageNotAvailableException extends RuntimeException {
```

Comment:
So why did you do this? Programmer must handle the situation of not having an external storage.